keyboards/exclusive/e7v1se/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
```

Comment:
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

Reply:
What is the purpose of adding this comment in the code @fauxpark? Don’t you need only a single option for this model?

Reply:
What is the purpose of adding this comment in the code @fauxpark? Don’t you need only a single option for this model?

Reply:
Yes, but I'd like to see these files adhere more closely to the template :)

Reply:
You asked to remove a bunch of boiler plate code in your previous comments for things that could be useful later, but in this case I don’t see the value. :-(
Is this the last comment before it can be merged?

Reply:
I've put it back. Can you merge this PR now @fauxpark?